tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/jvm/ObjectSubtypingTest.java

```diff
@@ -58,6 +58,19 @@ public void testCastingRuntimeError() {
         BRunUtil.invoke(compileResult, "testCastingRuntimeError");
     }
 
+    @Test(expectedExceptions = BLangRuntimeException.class,
```

Comment:
We can write these tests also in Ballerina. See https://github.com/ballerina-platform/ballerina-lang/blob/02b2ec6c18ce25569748d7f904055a2ab03f85a4/tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field.bal#L149.